src/main/java/org/squiddev/cobalt/Print.java

```diff
@@ -83,6 +83,50 @@
 		null,
 	};
 
+	public static final String[] OPNAMES_52 = {
```

Comment:
There's both a `Print.OPNAMES_52` and `Print52` (including `OPNAMES`). What's the correct one here?

Reply:
Oops, this got left in while trying to fix `Print`. `Print[52].OPNAMES` should be the correct one in both classes - I'll fix this soon. The `Print` code does have to be cleaned up a bit anyway - it was kind of hacked together to make tests work.